### ISSUE TITLE
Clean up PB benchmark examples

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#11401]
|[<img alt="teratorn's avatar" src="https://avatars.githubusercontent.com/u/154069?s=50" width="50" height="50">](https://github.com/teratorn)| @teratorn reported|
|-|-|
|Trac ID|trac#284|
|Type|defect|
|Created|2003-09-25 00:14:49Z|

Attachments:

* [pbbenchserver.py.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/7f3/7f35419a058e19d2b75e962dba149bddedec7606/6450fef4c1c548c81cdf25e5d4e7fb1f0b4bf7df.diff) (1910 bytes) - added by teratorn on 2003-09-25 00:14:49Z - 
* [pbbenchclient.py.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/7f3/7f35419a058e19d2b75e962dba149bddedec7606/5fb187edcdd5e0713b0d1a0dfbee1a78863648b6.diff) (1606 bytes) - added by teratorn on 2003-09-25 00:15:09Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__284 284
type__defect defect
reporter__teratorn teratorn
priority__high high
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__ 
keywords__ 
time__1064448889000000 1064448889000000
changetime__1467574515022383 1467574515022383
version__ 
owner__ 
cc__teratorn
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="teratorn's avatar" src="https://avatars.githubusercontent.com/u/154069?s=50" width="50" height="50">](https://github.com/teratorn)|@teratorn commented|
|-|-|

```
#!html
<pre>
pbbenchserver.py and pbbenchclient.py need to use new cred and new PB stuff.

</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)|@itamarst commented|
|-|-|

```
#!html
<pre>
Damn, I duplicated your work. Go bug glyph about cvs and
then rewrite the PB howtos, maybe? We will all say you are
the coolest person ever.

</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|GitHub commented|
|-|-|

In changeset 502f3a1e2cc125d214b6b7df5b173aa91fc7f966
```
#!CommitTicketReference repository="" revision="502f3a1e2cc125d214b6b7df5b173aa91fc7f966"
Merge pull request #284 from twisted/8548-rodrigc-thread: Fix test_win32events.py on Python3

Author: rodrigc
Reviewer: adiroiban
Fixes: #8548
```
